lib/config/fileconf.go

```diff
@@ -375,6 +380,10 @@ func MakeSampleFileConfig(flags SampleFlags) (fc *FileConfig, err error) {
 	if flags.ACMEEnabled {
 		p.ACME.EnabledFlag = "yes"
 		p.ACME.Email = flags.ACMEEmail
+		p.PublicAddr = utils.Strings{
```

Comment:
This means that the cluster name will have to be a valid domain name and have DNS record/be reachable?

Reply:
yes

Reply:
Gotcha. If this is an expectation we have (as opposed to, say, letting them provide public addr separately via a flag and set cluster name to whatever), then lgtm.